### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by rushikeshjadhav**
To reflect correct switch to build local and example latest branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ git clone https://github.com/xcp-ng-rpms/xapi.git
 # ... Here add your patches ...
 
 # Build.
-/path/to/run.py -b 7.5 --buildlocal xapi/ --rm
+/path/to/run.py -b 7.6 -l xapi/ --rm
```
Comment (stormi) [inline README.md]: I'd prefer --build-local (which is then mentioned to in the next paragraph)

Comment (rushikeshjadhav) [reply README.md]: --build-local is also fine. 

Currently it is `--buildlocal` which errors as `unrecognized arguments`

Comment (stormi) [reply README.md]: Yeah, I understood that :). Can you amend your commit and force-push it?

Comment (rushikeshjadhav) [reply README.md]: Hmm.. Should we be changing all `7.5` to `7.6`?

Comment (stormi) [reply README.md]: You can if you wish so (there will also be changes in the help of run.py itself)

**Pull request merged**